Makefile

```diff
@@ -56,7 +56,7 @@ DEFINES += NDEBUG
 
 # U2F
 DEFINES   += HAVE_U2F HAVE_IO_U2F
-DEFINES   += U2F_PROXY_MAGIC=\"BOIL\"
+DEFINES   += U2F_PROXY_MAGIC=\"~SOL\"
```

Comment:
One of our community members pointed out that we still had the default value here.  I couldn't find any guidance on choosing these values in the documentation.  Is this OK?